richardcoffey/minecraft-website
# Formatting and indention

This PR re-formats the `my-channel.html` file and introduces proper indention.

Steps to test:
* Render `my-channel.html`
* Ensure that styling and page display doesn't change
* Ensure that indention and formatting of `my-channel.html` file is correct.
